python/ray/air/checkpoint.py

```diff
@@ -259,7 +277,7 @@ def uri(self) -> Optional[str]:
             return self._uri
 
         if self._local_path and Path(self._local_path).exists():
-            return "file://" + self._local_path
+            return "file://" + str(self._local_path)
```

Comment:
Why do we need to call `str` here? Isn't `self._local_path` already a `str`?

Reply:
we guard against it from being set to `Path` by mistake - we could achieve the same effect by doing `return f"file://{self._local_path}"` which I suppose would be even better